.docker/messenger.yaml

```
@@ -0,0 +1,8 @@
+framework:
+    messenger:
+        transports:
+            pimcore_core: '%env(MESSENGER_TRANSPORT_DSN)%pimcore_core'

```

```suggestion
            pimcore_core: 'amqp://rabbitmq:5672'
```
For me it would be fine to just have it hardcoded here ... keeps things a bit simpler 😊

---

Changed to `amqp://rabbitmq:5672/%2f/pimcore_core` - so each queue is seperated:

```
$ docker compose exec rabbitmq rabbitmqctl list_queues
Timeout: 60.0 seconds ...
Listing queues for vhost / ...
name    messages
pimcore_maintenance     0
pimcore_image_optimize  0
pimcore_scheduled_tasks 0
pimcore_core    0
pimcore_asset_update    2
```